src/overlays/actors/ovl_En_Fr/z_en_fr.h

```diff
@@ -6,9 +6,57 @@
 
 struct EnFr;
 
+typedef void (*EnFrActionFunc)(struct EnFr*, struct GlobalContext*);
+typedef void (*EnFrBlinkFunc)(struct EnFr*);
+
+typedef enum {
+    /* 00 */ FROG_YELLOW,   // Middle
+    /* 01 */ FROG_BLUE,     // Front Left
+    /* 02 */ FROG_RED,      // Front Right
+    /* 03 */ FROG_PURPLE,   // Back Left
+    /* 04 */ FROG_WHITE     // Back Right
+} frogType;
```

Comment:
struct and enum are written as `FrogType`

Reply:
I added two more enums, let me know if those are named properly